### PR TITLE
Add all matroska tags

### PR DESCRIPTION
**Pull request by invliD**
I found some more tags I need in addition to #135, so I figured I'd just add them all.

### CODE REVIEW

File: elementtable.go [hunk 1]
```diff
@@ -26,91 +26,235 @@ type elementDef struct {
...
+	ElementCueDuration:                 elementDef{[]byte{0xB2}, DataTypeUInt, false},
+	ElementCueTime:                     elementDef{[]byte{0xB3}, DataTypeUInt, false},
+	ElementSamplingFrequency:           elementDef{[]byte{0xB5}, DataTypeFloat, false},
+	ElementChapterAtom:                 elementDef{[]byte{0xB6}, DataTypeMaster, false},
+	ElementCueTrackPositions:           elementDef{[]byte{0xB7}, DataTypeMaster, false},
+	ElementFlagEnabled:                 elementDef{[]byte{0xB9}, DataTypeUInt, false},
+	ElementPixelHeight:                 elementDef{[]byte{0xBA}, DataTypeUInt, false},
+	ElementCuePoint:                    elementDef{[]byte{0xBB}, DataTypeMaster, false},
+	ElementCRC32:                       elementDef{[]byte{0xBF}, DataTypeBinary, false},
+	ElementLaceNumber:                  elementDef{[]byte{0xCC}, DataTypeUInt, false},
```
Comment (at-wat) [inline elementtable.go:236]: Seems be dropped by matroska v2

File: elementtable.go [hunk 2]
```diff
@@ -26,91 +26,235 @@ type elementDef struct {
...
+	ElementCueReference:                elementDef{[]byte{0xDB}, DataTypeMaster, false},
+	ElementVideo:                       elementDef{[]byte{0xE0}, DataTypeMaster, false},
+	ElementAudio:                       elementDef{[]byte{0xE1}, DataTypeMaster, false},
+	ElementTrackOperation:              elementDef{[]byte{0xE2}, DataTypeMaster, false},
+	ElementTrackCombinePlanes:          elementDef{[]byte{0xE3}, DataTypeMaster, false},
+	ElementTrackPlane:                  elementDef{[]byte{0xE4}, DataTypeMaster, false},
+	ElementTrackPlaneUID:               elementDef{[]byte{0xE5}, DataTypeUInt, false},
+	ElementTrackPlaneType:              elementDef{[]byte{0xE6}, DataTypeUInt, false},
+	ElementTimecode:                    elementDef{[]byte{0xE7}, DataTypeUInt, false},
+	ElementTimeSlice:                   elementDef{[]byte{0xE8}, DataTypeMaster, false},
```
Comment (at-wat) [inline elementtable.go:247]: Seems be dropped by matroska v2

File: elementtable.go [hunk 3]
```diff
@@ -26,91 +26,235 @@ type elementDef struct {
...
+	ElementMaxCache:                    elementDef{[]byte{0x6D, 0xF8}, DataTypeUInt, false},
+	ElementChapterSegmentUID:           elementDef{[]byte{0x6E, 0x67}, DataTypeBinary, false},
+	ElementChapterSegmentEditionUID:    elementDef{[]byte{0x6E, 0xBC}, DataTypeUInt, false},
+	ElementTrackOverlay:                elementDef{[]byte{0x6F, 0xAB}, DataTypeUInt, false},
+	ElementTag:                         elementDef{[]byte{0x73, 0x73}, DataTypeMaster, false},
+	ElementSegmentFilename:             elementDef{[]byte{0x73, 0x84}, DataTypeString, false},
+	ElementSegmentUID:                  elementDef{[]byte{0x73, 0xA4}, DataTypeBinary, false},
+	ElementChapterUID:                  elementDef{[]byte{0x73, 0xC4}, DataTypeUInt, false},
+	ElementTrackUID:                    elementDef{[]byte{0x73, 0xC5}, DataTypeUInt, false},
+	ElementAttachmentLink:              elementDef{[]byte{0x74, 0x46}, DataTypeUInt, false},
```
Comment (at-wat) [inline elementtable.go:188]: Seems be dropped by matroska v4

**Review by at-wat — COMMENTED** [review]
Overall looks good to me. It would be better to add comments for deprecated elements.

**Comment by invliD** [comment]
Yeah, I intentionally added the deprecated ones (specifically all those that were included in at least one version of the standard), because I actually have older files that include some of these elements, and they wouldn't be parseable without them defined here. I'll add the comments to those elements.

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**